src/xdsl/irdl.py

```diff
@@ -242,7 +245,7 @@ class SingleBlockRegionDef(RegionDef):
 
 
 @dataclass(init=False)
-class AttributeDef:
+class AttributeDef(Attribute):
```

Comment:
This breaks the pytests, due to `Attribute` being a frozen class. Is this required for VSCode, or can this be reverted?